### DISCUSSION TITLE
Support for decoding H.265 in FLV with new codecid=12 for Chinese CDN manufacturers in FFmpeg 7.0

### DISCUSSION THREAD

**Discussion by xinyangbest** [asticode/go-astiav#83] [Q&A] [answered]
How can the source code of FFmpeg version 7.0 support compilation to adapt to the new codecid = 12 method added by Chinese CDN manufacturers and enable decoding of H.265 video in FLV format?

**Comment by asticode** [comment, marked as answer]
This is unfortunately a question you would have to ask the `FFMpeg` dev team as it is linked to `FFMpeg` compilation and not this library 👍
